frontend/src/components/Utils/time.js

```diff
@@ -35,3 +35,26 @@ export function timestampToDateFormatted(msTimestamp, timezone='America/New_York
   const formatted = date.toLocaleDateString('en-US', formatOptions);
   return formatted;
 }
+
+/** 
+ * Format a timestamp (in milliseconds) into a pretty string.
+ * 
+ * @param {int} msTimestamp 
+ * @param {string} timezone 
+ * @returns {string} Time formatted into a string like 
+ * "Monday, January 19, 1970, 02:48 AM"
+ */
+export function timestampToFormatted(msTimestamp, timezone = "America/New_York") {
+  let date = new Date(msTimestamp);
+  let formatOptions = { 
+    weekday: 'long', 
+    year: 'numeric', 
+    month: 'long', 
+    day: 'numeric',  
+    hour: 'numeric', 
+    minute: '2-digit', 
+    timeZone: timezone
+  };
+  let formatted = date.toLocaleString('en-US', formatOptions);
+  return formatted;
```

Comment:
Maybe just return
```
date.toLocaleString('en-US', formatOptions)
```
here instead of putting it in a variable first?

Consider declaring the unchanging variables in this function with const instead of let, as well.